README.md

```diff
@@ -37,6 +37,10 @@ Also it is very important to **use a external power supply**. USBs usually do no
 > ![schematics to wire the board](./assets/Wiring-with-usb-ttl.png)<br />
 > Diagram to wire the board
 
+## Pin Notes
```

Comment:
What do you think of moving this section into the [about page](https://github.com/raphaelbs/esp32-cam-ai-thinker/blob/master/docs/about-esp32-cam.md)?